wandb/sdk/wandb_init.py

```diff
@@ -186,6 +191,16 @@ def setup(self, kwargs) -> None:
             if settings._jupyter:
                 self._jupyter_setup(settings)
 
+            if (
+                not settings._attach_id
+                and multiprocess.is_likely_multiprocessing_process()
+            ):
+                wandb.termwarn(
+                    "Detected wandb.init() called from multiprocessing process: See http://wandb.me/client-"
```

Comment:
TODO: fix this link

Reply:
yes,  i have a new module that consolidates messages related to concurrency... coming soon